src/ServiceProviders/RestAPI.php

```diff
@@ -64,6 +65,7 @@ class RestAPI implements ServiceProvider
         FormActions::class,
         GetMigrations::class,
         RunMigration::class,
+        ProductRecommendationsRoute::class,
```

Comment:
This file is, um, a bit out of control. Hahah! The variable name is `$reportRoutes` and it's just a dumping ground for endpoints. 😅 

Please use the `Promotions/ServiceProvider` instead. The `src/ServiceProviders` folder should ultimately be removed.

Reply:
Got it yea, Resolved: https://github.com/impress-org/givewp/pull/6797/commits/b57b1ca8a13be20e0329dd1ffa87abcd243f2823